### PR TITLE
fix: empty value written to cache on error

### PR DESCRIPTION
**Pull request by GalvinGao**

### CODE REVIEW

File: internal/pkg/cache/set.go [hunk 1]
```diff
@@ -43,11 +44,15 @@ func (c *Set) Get(key string, dest interface{}) error {
 	} else {
 		r = reflect.ValueOf(result)
 	}
+	fmt.Println(result, dest)
+	fmt.Println(reflect.TypeOf(result), reflect.TypeOf(dest))
+	fmt.Println(r.Type(), reflect.TypeOf(dest))
 	reflect.ValueOf(dest).Elem().Set(r)
```
Comment (AlvISsReimu) [inline internal/pkg/cache/set.go:50]: are these output removable

**Review by AlvISsReimu — CHANGES_REQUESTED** [review]
some `fmt.Println` can be removed i guess

File: internal/pkg/cache/set.go (hunk 1 above)
Comment (GalvinGao) [reply internal/pkg/cache/set.go]: resolved in 25e820673bce6039989fea3f5a1a437486a8ad44

**Review by AlvISsReimu — APPROVED** [review]
lgtm

**Pull request merged**